sdk/textanalytics/tests.yml

```diff
@@ -12,15 +12,8 @@ extends:
             - $(sub-config-text-analytics-azure-cloud-test-resources)
           MatrixReplace:
             - TestSamples=.*/true
-        UsGov:
```

Comment:
Did you intend to remove these configs?

Reply:
I'm forgetting, aren't they not necessary anymore?

Reply:
If you override one config you need to list all them in your custom config list.